src/packtPublishingFreeEbook.py

```diff
@@ -128,6 +128,14 @@ def __create_http_session(self):
             raise requests.exceptions.RequestException(message)
         logger.info("Session created, logged in successfully!")
 
+    def __exists_book(self, title):
```

Comment:
Why is it a private method? Can we have a docstring for it?

Reply:
because it just use once and action is inefficient. (but I haven't an idea for perf tune up yet)